js/package.json

```diff
@@ -2,73 +2,103 @@
   "name": "nglview-js-widgets",
   "version": "2.7.7",
   "description": "nglview-js-widgets",
-  "author": "Hai Nguyen <hainm.comp@gmail.com>, Alexander Rose <alexander.rose@weirdbyte.de>",
-  "license": "MIT",
-  "main": "lib/index.js",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/arose/nglview.git"
-  },
+  "keywords": [
+    "ipython",
+    "ipywidgets",
+    "jupyter",
+    "jupyterlab",
+    "jupyterlab-extension",
+    "molecular graphics",
+    "molecular structure",
+    "science",
+    "widgets"
+  ],
+  "homepage": "https://github.com/nglviewer/nglview/js",
   "bugs": {
-    "url": "https://github.com/arose/nglview/issues"
+    "url": "https://github.com/nglviewer/nglview/issues"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Alexander Rose",
+    "email": "alexander.rose@weirdbyte.de"
   },
+  "contributors": [
+    {
+      "name": "Hay Nguyen",
```

Comment:
Should be `Hai Nguyen`, not `Hay Nguyen`

Reply:
Apologies for the spelling mistake!